src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -230,14 +238,11 @@ def __call__(
             if latents.shape != latents_shape:
                 raise ValueError(f"Unexpected latents shape, got {latents.shape}, expected {latents_shape}")
         latents = latents.to(self.device)
+        latents = self.scheduler.scale_initial_noise(latents)
```

Comment:
Nice this makes sense to me!

Reply:
I don't think this works correctly at the moment because the sigmas are changed when doing `self.scheduler.set_timesteps`